web/src/containers/Admin/Fees/index.js

```diff
@@ -60,14 +61,19 @@ class BlockchainTransaction extends Component {
 							title="Fees"
 							style={{ textAlign: 'center' }}
 						>
-							{Object.entries(data.fees).map(([currency, amount], index) => (
-								<div
-									key={index}
-									className="list-group-item list-group-item-action"
-								>
-									{currency.toUpperCase()} : {formatCurrency(amount)}
-								</div>
-							))}
+							{Object.entries(data.fees).map(([currency, amount], index) => {
+								const increment_unit = this.props.coins?.[currency]
```

Comment:
I think it's better to destructure the coins at the top of the render section
`const { coins } = this.props`